### ISSUE TITLE
Please tag releases so they can be tracked downstream by package managers/repositories

### ISSUE THREAD

**Issue by luzpaz** [sk1project/sk1-wx#234]
https://help.github.com/en/articles/creating-releases

**Comment by Joermungand** [comment]
These are RC snapshots, not releases. Package mangers can keep an RSS feed like I do (for AUR). As far as I’m concerned, there are no actual releases as of yet, so no need for tags.

**Comment by luzpaz** [comment]
They can be tagged as pre-releases  
![pre-release](https://help.github.com/assets/images/help/releases/prerelease_checkbox.png)

**Comment by Joermungand** [comment]
> They can be tagged as pre-releases

Question is: *need* they?

**Comment by luzpaz** [comment]
Answer: what's the harm ?  
Like I said, it can be used to inform downstream. To elaborate, there are package tracking repositories or services like freshcode.club and wikidata.org etc... along with https://packages.aosc.io/packages/sk1 and other builtin scripts that run to check tags/releases of upstream repositories.  

Per https://repology.org/project/sk1/badges

**Comment by Joermungand** [comment]
The harm is in diverting developer resources from their actual purposes. Builtin scripts (made by humans) can be adapted to check for Git commits if need be (although it is doubtful if such snapshots should end up in repositories, seeing that they are nightly ‘releases’) – yay on Arch does a pretty good job at that so it may serve as an example script. The fact that scripts do not provide for such cases is not reason enough to have the developers do the extra work and call them to attention if they don’t. At the end of the day, a script is a script: it will only mirror its designer’s thinking, not more.

**Comment by luzpaz** [comment]
fair point. Something to note though, that this is an emerging trend that looks very well established across the package ecosystem. It's like having a README.md in parallel to a README.txt People/scripts (what have you) will be looking in the releases/tags section to see where the project is at. Just something to be mindful of. Especially when it comes to visibility. Most people think of Scribus as the leading edge software in this genre.
